app_dart/lib/src/service/content_aware_hash_service.dart

```diff
@@ -112,4 +127,102 @@ interface class ContentAwareHashService {
     // Fail
     return null;
   }
+
+  /// Finds the hash status for [job] and updates any tracking docs.
+  Future<MergeQueueHashStatus> processWorkflowJob(WorkflowJobEvent job) async {
+    final hash = await hashFromWorkflowJobEvent(job);
+    if (hash == null) return MergeQueueHashStatus.unknown;
```

Comment:
Do we expect this to happen as part of normal workflows?

If no, let's add a log.warn I think.

Reply:
`hashFromWorkflowJobEvent()` is thrown _all workflow jobs_; including non-content-aware-hash, queueing, etc.  I will change this to "ignoreJob" and then add an error with a warn so we can monitor the logs.

```dart
      case 'workflow_job':
        try {
          final job = workflow_job.WorkflowJobEvent.fromJson(
            json.decode(webhook.payload) as Map<String, Object?>,
          );
          log.debug('workflow_job: $job');
          await scheduler.processWorkflowJob(job);
```